src/Button/Button.stories.tsx

```diff
@@ -2,23 +2,10 @@ import {EyeClosedIcon, EyeIcon, SearchIcon, TriangleDownIcon, XIcon} from '@prim
 import {Meta} from '@storybook/react'
 import React, {useState} from 'react'
 import {Button, ButtonProps, IconButton} from '.'
-import {BaseStyles, ThemeProvider} from '..'
 import Box from '../Box'
 
 export default {
   title: 'Composite components/Button',
-
-  decorators: [
-    Story => {
-      return (
-        <ThemeProvider>
-          <BaseStyles>
-            <Story />
-          </BaseStyles>
-        </ThemeProvider>
-      )
-    }
-  ],
```

Comment:
We do this in every story, which creates nested ThemeProviders! Switching theme at the top level doesn't go all the way down :/

Needs some clean up after this PR is merged.